fdbserver/workloads/PerpetualWiggleStatsWorkload.actor.cpp

```diff
@@ -158,6 +158,7 @@ struct PerpetualWiggleStatsWorkload : public TestWorkload {
 		MoveKeysLock lock = wait(takeMoveKeysLock(cx, UID())); // force current DD to quit
 		bool success = wait(IssueConfigurationChange(cx, "storage_migration_type=disabled", true));
 		ASSERT(success);
+		wait(delay(30.0)); // make sure the DD has already quit before the test start
```

Comment:
How about just checking `moveKeysLockOwnerKey` has changed after line 158? Could register a watch before line 158.

Reply:
`setDDMode` ,`IssueConfigurationChange`, and a restarted but disabled DD all change the `moveKeysLockOwnerKey`.
It's hard to say the restarted DD change the key before or after a watch is set, unless we change `setDDMode()` to return the lock it set (let's say it's `La`), then read the lock again (`Lb`) and compare with `La`. 
If they are different return immediately otherwise wait on a watch.